lms/envs/aws.py

```diff
@@ -331,7 +331,6 @@
                             service_variant=SERVICE_VARIANT)
 
 COURSE_LISTINGS = ENV_TOKENS.get('COURSE_LISTINGS', {})
-VIRTUAL_UNIVERSITIES = ENV_TOKENS.get('VIRTUAL_UNIVERSITIES', [])
```

Comment:
@stvstnfrd do we need META_UNIVERSITIES? or can that also get deleted here.

Reply:
@feanil I see no other references.

Removed in 5d6e6a42833158209386866ca6804dd9d93f9a69